content/messaging/agent-chat-api/v3.3/index.mdx

```diff
@@ -118,6 +118,42 @@ The **File** event informs about an uploaded file.
 </Code>
 </Section>
 
+## Form
```

Comment:
Nothing links here - there is a list of event types higher up on this page where the new event should be referenced